### DISCUSSION TITLE
Elsa3: Default activity join behaviour "wait any" possible?

### DISCUSSION THREAD

**Discussion by JochenSteinbrenner** [elsa-workflows/elsa-core#4483] [Q&A] [answered]
Good morning,

is it possible to change / configure the default activity join behaviour onto application level?
At the moment, the default behaviour is "wait all" which causes me to add many join activities with behaviour  "wait any" into my
workflows.

For me, normally all activies should be based on "wait any". Only a view cases do "wait all".

Thank you for your help

with kind regards
Jochen Steinbrenner

**Comment by sfmskywalker** [comment]
This is not possible at the moment, but you're the second person who's asked about it so I will create an issue for it and put it on the roadmap 👍🏻

**Reply by sfmskywalker** [reply, marked as answer]
Issue created [here](https://github.com/elsa-workflows/elsa-core/issues/4508).
